ark/phenotyping/som_utils.py

```diff
@@ -619,10 +680,7 @@ def cluster_pixels(fovs, base_dir, pre_dir='pixel_mat_preprocessed',
     misc_utils.verify_in_list(provided_fovs=fovs,
                               subsetted_fovs=io_utils.remove_file_extensions(files))
 
-    # precompute row sums for each fov (more efficient in Python than R)
-    print("Normalizing row sums and removing rows that sum to 0")
     weights = feather.read_dataframe(os.path.join(base_dir, weights_name))
-    preprocess_row_sums(fovs, weights.columns.values, base_dir, pre_dir)
 
     # ensure the norm vals columns are valid indexes
```

Comment:
In the two function calls to misc_utils.verify_same_elements below, why does one have enforce_order=True and the other one doesn't? I would assume that both norm_vals and pixel_weights should have markers in the same order.